README.md

```diff
@@ -1,63 +1,87 @@
 # ☄️ Effector
 
-Reactive state manager
+### Reactive state manager
 
 [![npm version](https://badge.fury.io/js/effector.svg)](https://badge.fury.io/js/effector) [ ![Codeship Status for zerobias/effector](https://app.codeship.com/projects/67f481f0-2c7e-0136-030e-1a8413355f0c/status?branch=master)](https://app.codeship.com/projects/288022) [![Build Status](https://semaphoreci.com/api/v1/zerobias/effector/branches/master/shields_badge.svg)](https://semaphoreci.com/zerobias/effector)
+[![Join the chat at https://
+.im/effector-js/community](https://img.shields.io/gitter/room/effector-js/community.svg?style=popout)](https://gitter.im/effector-js/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+![PRs welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg)
+![License](https://img.shields.io/npm/l/effector.svg?colorB=brightgreen&style=popout)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-## Table of Contents
+## Introduction
 
-- [Motivation](#motivation)
-- [Installation](#installation)
-- [About](#about)
-  - [Hello world with events and nodejs](#hello-world-with-events-and-nodejs)
-  - [Storages and events](#storages-and-events)
-- [Demo](#demo)
-- [Wiki](#wiki)
-- [Usage](#usage)
-  - [Domain hooks](#domain-hooks)
-- [Typings](#typings)
-- [API](#api)
-- [Core types](#core-types)
-- [Contributors](#contributors)
-- [License](#license)
+Effector is an effective multi store state manager for Javascript apps __(React/Vue/Node.js)__, that allows you to manage data in complex applications without the risk of inflating the monolithic central store, with clear control flow, good support types and high capacity API. Effector supports both TypeScript and Flow type annotations out of the box.
```

Comment:
* Extra comma before *that*
* Not *good support types*, but *good type support*